test/search.cc

```diff
@@ -198,11 +211,16 @@ void search(valhalla::baldr::Location location, size_t result_count, int reachab
   valhalla::Location pbf;
   PathLocation::toPBF(location, &pbf, reader);
   location = PathLocation::fromPBF(pbf);
+  const auto costing = create_costing();
 
-  const auto results = Search({location}, reader);
+  const auto results = Search({location}, reader, costing);
   if (results.empty() && result_count == 0)
     return;
+
+  LOG_WARN("indexing map");
+  // TODO Exception raised here _Map_at
   const auto& p = results.at(location);
```

Comment:
Tests now throw `_Base_at` here which I assume means `location` is not in the result.

Reply:
that happens when it doesnt find a result. its basically the side effect of not finding any edges that match the request.